codebase/src/java/disco/org/openlvc/disco/configuration/RprConfiguration.java

```diff
@@ -491,7 +491,9 @@ private void loadDefaultModules()
 			"hla/rpr2/RPR-Enumerations_v2.0.xml",
 			"hla/rpr2/RPR-Physical_v2.0.xml",
 			"hla/rpr2/RPR-SIMAN_v2.0.xml",
-			"hla/rpr2/RPR-Warfare_v2.0.xml"
+			"hla/rpr2/RPR-Warfare_v2.0.xml",
+			"hla/rpr2/RPR-Aggregate_v2.0.xml",
+			"hla/rpr2/RPR-SE_v2.0.xml",
```

Comment:
Are these extra files already included in the codebase?

Reply:
Yeah... I don't know why they weren't included (perhaps we just never used anything in them?)

Reply:
These shouldn't be needed by Disco and should be removed. Any module that is not 100% required (because there is a mapper that uses data from it) should be excluded. If other federates/applications need these models they can add them within the scope of their applications.